src/zenml/zen_stores/rest_zen_store.py

```diff
@@ -310,21 +301,12 @@ def update_stack(self, name: str, stack: StackWrapper) -> Dict[str, str]:
             name: The original name of the stack.
             stack: The new stack to use in the update.
 
-        Returns:
-            metadata dict for telemetry or logging.
-
         Raises:
             ValueError: If a dict is not returned from the API.
         """
```

Comment:
If you've deleted the raising of a `ValueError`, then this should be deleted as well.

Not quite sure why you deleted that check, though.